c_binding/include/libstoragemgmt/libstoragemgmt.h

```diff
@@ -224,63 +439,164 @@ int LSM_DLL_EXPORT lsm_job_status_ss_get(lsm_connect *c,
                                          lsm_fs_ss ** ss, lsm_flag flags);
 
 /**
- * Frees the resources used by a job.
- * @param[in] conn          Valid connection pointer
- * @param[in] job_id        Job ID
- * @param[in] flags         Reserved for future use, must be zero.
- * @return Error code as enumerated by \ref lsm_error_number.
- * @retval LSM_ERR_OK on success.
+ * lsm_job_free - Frees the resources used by a job.
+ *
+ * Version:
+ *      1.0
+ *
+ * Description:
+ *      Frees the resources used by a job.
+ *
+ * @conn:
+ *      Valid connection pointer
+ * @job_id:
+ *      String. Job ID
```

Comment:
Note: The memory used for the job id will be freed during this call and job id will be set to NULL.